src/common/res/features/goal-indicator/main.js

```diff
@@ -0,0 +1,106 @@
+(function poll() { 
+  // Waits until an external function gives us the all clear that we can run (at /shared/main.js)
+  if ( typeof ynabToolKit !== "undefined"  && ynabToolKit.pageReady === true ) {
+  
+    ynabToolKit.featureOptions.goalIndicator = true;
+    ynabToolKit.goalIndicator = function ()  { // Keep feature functions contained within this
+    	var entityManager = ynab.YNABSharedLib.getBudgetViewModel_AllAccountTransactionsViewModel()._result.getEntityManager();
+
+    	function addIndicator (element, inticator) {
+    		var budgetedCell = $(element).find(".budget-table-cell-budgeted");
+    		if (budgetedCell.has(".goal-indicator").length == 0) {
```

Comment:
@iamnafets I use budgetCel in the 12 line too, so this is not good. Why do you want so?